DEPLOYMENT.md

```diff
@@ -0,0 +1,35 @@
+# Deployment
+
+Nuts Knooppunt is provided as a single docker image that should be deployed within a participating organisations
+network. We will update this guide as we implement the different generic functions.
+
+Knooppunt is available from the Github Container Registry.
+
+https://github.com/nuts-foundation/nuts-knooppunt/pkgs/container/nuts-knooppunt
+
+Knooppunt embeds a Nuts Node which is disabled by default.
```

Comment:
```suggestion
Knooppunt embeds a Nuts Node which is enabled by default.
```

Reply:
We wrote in #90 that it should be disabled by default. But I do see we have configured it to be enabled now. Shouldn't we disable it by default until the implementor decides to use it?

Reply:
Sure, but we need to change that.

Reply:
Done!